ruby/example_code/rds/list_parameter_groups.rb

```diff
@@ -0,0 +1,28 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# snippet-start:[rds.ruby.listParameterGroups]
+require "aws-sdk-rds"  # v2: require 'aws-sdk'
+
+# List all Amazon Relational Database Service (Amazon RDS) parameter groups.
+#
+# @param rds_resource [Aws::RDS::Resource] An SDK for Ruby Amazon RDS resource
+# @return [Array, nil] List of all parameter groups or nil if error
```

Comment:
@return [Array, nil] List of all parameter groups, or nil if error.

Reply:
Done.